odoo/tests/common.py

```diff
@@ -702,15 +702,24 @@ def executable(self):
                     return bin_
 
         elif system == 'Windows':
-            # TODO: handle windows platform: https://stackoverflow.com/a/40674915
-            pass
+            bins = [
+                '%ProgramFiles%\Google\Chrome\Application\chrome.exe',
+                '%ProgramFiles(x86)%\Google\Chrome\Application\chrome.exe',
+                '%LocalAppData%\Google\Chrome\Application\chrome.exe',
+            ]
+            for bin_ in bins:
+                bin_ = os.path.expandvars(bin_)
+                if os.path.exists(bin_):
+                    return bin_
 
         raise unittest.SkipTest("Chrome executable not found")
 
     def _spawn_chrome(self, cmd):
-        if os.name != 'posix':
-            return
-        pid = os.fork()
+        if os.name == 'nt':
+            proc = subprocess.Popen(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+            pid = proc.pid
+        else:
+            pid = os.fork()
```

Comment:
I think this creates a more complicated and contorted codepath, I prefer when there was just `return proc.pid` in the nt branch and the posix code was untouched.

Reply:
I don't use just `return proc.pid` because starting from 14.0 there is additional code that is necessary for `nt` branch too.
For 13.0 it can be as you suggested, but starting from 14.0 not.